CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.7.5]
+### Fixed
+- Add support for Finshark (Open Banking). 
```

Comment:
```suggestion
- Add support for Open Banking
```

Our payment method is Open Banking. The fact that we use Finshark as engine is just for more info to the merchants.